input/fsh/instances/condition-infection-present-during-pregnancy-jada-ann-quinn.fsh

```diff
@@ -4,7 +4,7 @@ Title: "Condition - Infection Present During Pregnancy - Jada Ann Quinn"
 Description: "Condition - Infection Present During Pregnancy: Jada Ann Quinn example"
 Usage: #example
 * category[us-core] = $condition-category#problem-list-item "Problem List Item"
-* category[infectionDuringPregnancy] = $loinc#72519-2 "Infections present AndOr treated during this pregnancy for live birth [US Standard Certificate of Live Birth]"
+* category[requiredCategory] = $loinc#72519-2 "Infections present AndOr treated during this pregnancy for live birth [US Standard Certificate of Live Birth]"
```

Comment:
Do these instances need to include this explicitly? Others do not include it (e.g., Chorioamniotis). Maybe I misunderstand the slicing

Reply:
Correct.